package.json

```diff
@@ -51,13 +51,13 @@
         "grunt": "1.3.0",
         "grunt-contrib-clean": "2.0.0",
         "grunt-contrib-compress": "1.6.0",
-        "grunt-contrib-concat": "1.0.1",
         "grunt-contrib-copy": "1.0.0",
         "grunt-contrib-uglify": "4.0.1",
```

Comment:
grunt-contrib-uglify should be removed from the dependencies

Reply:
Resolved in d6090f6.